dockernet/src/init_chain.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
+
```

Comment:
Can we put all the other sh files 
(build.sh, config.sh, start_network.sh) in src file for sanity?

Reply:
Ok @mattverse !